go.mod

```diff
@@ -1,4 +1,4 @@
-module main
+module firehose-logs
```

Comment:
This `go.mod` file is redundant here. Please add `go.mod` to `cloudformation` and `eventbridge-lambda` and name the modules according to this pattern `github.com/logzio/<<repo-name>>/<<path-to-module>>` see this example: https://github.com/logzio/prometheus-alerts-migrator/blob/master/go.mod#L1

Reply:
I've updated the module path in the `go.mod` file as suggested. However, I'd like to discuss the recommendation to add separate `go.mod` files for each Lambda function. Given that our Lambda functions share a significant amount of code and dependencies, I believe maintaining a single `go.mod` at the project root simplifies dependency management, reduces maintenance overhead, and enhances project cohesion. This approach seems more manageable for our current project structure, ensuring consistent dependency versions across all components and streamlining our build and deployment processes. 
Could we explore the benefits of separate `go.mod` files for our specific use case, considering these points?

Reply:
If you think we need to manage the dependencies of all of the function included (now and in the future) together, I Can accept it. Its fine if they share significant amount of code and dependencies. Anyway rename the module to the pattern I mentioned